bundles/automation/org.eclipse.smarthome.automation.core.test/src/test/java/org/eclipse/smarthome/automation/core/internal/RuleEngineTest.java

```diff
@@ -41,18 +41,19 @@
 
     Logger log = LoggerFactory.getLogger(RuleEngineTest.class);
 
-    private RuleEngine createRuleEngine() {
+    private RuleEngine createRuleEngine() throws Exception {
```

Comment:
I comment this only here on the first occurrence (as it is used multiple times):
As it is only a test I don't care very much, but I don't think it is good practice to throw the type "Exception".
There are so many different exceptions that normally are not handled all the same way.

But if it is only for testing...